### PR TITLE
VB-1263 Add creation date and update date to migration api

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

allows nomis to send create and update date to be passed into the VSiP system 

## What is the intent behind these changes?

Allows users of visip to know when the visit was created rather than the migrated time in VSIP

### CODE REVIEW

**Comment by AledWEvans** [comment]
I have put this to WIP as we have not had a convo with SYSCON yet

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt [hunk 1]
```diff
@@ -100,4 +102,20 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
     "SELECT CASE WHEN (COUNT(v) = 1) THEN TRUE ELSE FALSE END  FROM Visit v WHERE v.reference = :bookingReference AND v.visitStatus = 'BOOKED' "
   )
   fun isValidBookingReference(bookingReference: String): Boolean
+
+  @Transactional
+  @Modifying
+  @Query(
+    "Update visit SET  modify_timestamp = :modifyTimestamp WHERE id=:id",
+    nativeQuery = true
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt:110]: We could change this from nativeQuery to a JPA query.

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt:110]: native could be much faster!

**Review by dhirajshettymoj — APPROVED** [review]
Looks good to me.

**Pull request merged**